dear_petition/petition/models.py

```diff
@@ -150,6 +182,8 @@ class OffenseRecord(PrintableModelMixin, models.Model):
     action = models.CharField(max_length=256)
     severity = models.CharField(max_length=256)
     description = models.CharField(max_length=256)
+    disposition = models.CharField(max_length=256, blank=True)
+    is_visible = models.BooleanField(default=True)
```

Comment:
I think these fields can be converted into properties as they are deriving data from the offense. It would look like this:

```python
@property
def disposition(self):
    if self.action == CHARGED:
        if offense.is_guilty_to_lesser():
            return VERDICT_GUILTY_TO_LESSER
        elif offense.is_responsible_to_lesser():
            return VERDICT_RESPONSIBLE_TO_LESSER
    return offense.verdict if offense.verdict else offense.disposition_method

@property
def is_visible(self):
    return not (self.action == CHARGED and offense.is_convicted_of_charged())
```